hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/AvroToRowDataConverters.java

```diff
@@ -233,7 +234,9 @@ private static AvroToRowDataConverter createTimestampConverter(int precision) {
               "Unexpected object type for TIMESTAMP logical type. Received: " + avroObject);
         }
       }
-      return TimestampData.fromInstant(instant);
+      Timestamp timestamp = new Timestamp(instant.toEpochMilli());
+      timestamp.setNanos(instant.getNano());
+      return TimestampData.fromTimestamp(timestamp);
```

Comment:
We should not add local time zone to this, you can just fix the tests.

Reply:
Because RowDataToAvroConverters has used local time zone,so this should  also use local time zone，otherwise read  mor  avro log is  8 hours ahead of the time .

Reply:
No, `RowDataToAvroConverters` uses the UTC timezone.

Reply:
I have  modified   from
 _((TimestampData) object).toInstant().toEpochMilli();_
to
 _((TimestampData) object).toTimestamp().toInstant().toEpochMilli();_

Reply:
You should not do that.

Reply:
I see，this pr  mainly used local timezone  to solve hive  8 hours  late issues,  so we  can close this pr.